### DISCUSSION TITLE
Issue while trying to package the VS code extension using `vsce`

### DISCUSSION THREAD

**Discussion by YHAMSTM** [eclipse-glsp/glsp#596] [Q&A] [answered]
Hello there,
I am still investigating GLSP project to create a VS Code Extension around it.
To do so, I tried the `glsp-vscode-integration` and working with it. Debugging is working correctly now, thanks to Eclipse's community help. However, it looks like the `vsce` packager does not support `yarn` workspace / lerna monorepo. 
Archive contains empty node_modules, and dependencies are not available from the VSIX package.
I did not find any script on the project to build the VSIX, so I tried to do:
- cd `example/workflow/extension` folder
- vsce package --yarn

The archive does not contain any monorepo dependencies...
![image](https://user-images.githubusercontent.com/97888392/158972361-6d633a84-c53a-4ff4-8c03-c447be3b26cc.png)

Is the packaging procedure documented anywhere ?
Thanks in advance!

**Comment by tortmayr** [comment, marked as answer]
Hi @hamitiya-STM,
the example in the `vscode-integration` was never intended for packing/publishing and a little bit of additional configuration is required
to achieve this. This usecase is also interesting for other adopters we should probably update the workflow example extension to properly support packaging. I have created #597 to track this.

As  you already noticed `vsce` doesn't work well with yarn in combination with lerna. To easiest way to solve this issue is to bundle the
workflow example extension with webpack. Then `vsce package --yarn` should work as intended.

I have already started working on the PR for #597. You can have a look at the WIP branch (https://github.com/eclipse-glsp/glsp-vscode-integration/commit/0e61fcf5413c283917ef89d3b983375351c038e9) to see the necessary changes.

**Reply by YHAMSTM** [reply]
I'll give it a try, many thanks!
